src/routes/stake/Unstake.svelte

```diff
@@ -77,11 +97,228 @@
 		isConverting.set(false);
 	}
 
+	const approveInFastUnstake = async (spender: Approve, amountE8s: bigint) => {
+		const approveResult: ApproveResult = await $canisters.nicpLedger.icrc2_approve({
+			spender,
+			fee: [],
+			memo: [],
+			from_subaccount: [],
+			created_at_time: [],
+			expires_at: [],
+			expected_allowance: [],
+			amount: amountE8s
+		} as ApproveArgs);
+
+		const status = handleApproveResult(approveResult);
+		if (!status.success) {
+			toasts.add(Toast.error(status.message));
+			isConverting.set(false);
+		}
+	};
+
+	const depositInFastUnstake = async (amountE8s: bigint) => {
+		const depositResult = await $canisters.icpswap.depositFrom({
+			fee: FEE,
+			token: CANISTER_ID_NICP_LEDGER,
+			amount: amountE8s - FEE
+		} as DepositArgs);
+		const key = Object.keys(depositResult)[0] as keyof IcpSwapResult;
+		switch (key) {
+			case 'ok':
+				break;
+			case 'err':
+				const error = depositResult[key];
+				const errorKey = Object.keys(error)[0] as keyof IcpSwapError;
+				console.log('[ICPSwap deposit] error:', error[errorKey]);
+				toasts.add(Toast.error('Failed to deposit nICP on ICPSwap. Please try again.'));
+				break;
+		}
+		return depositResult;
+	};
+
+	const swapInFastUnstake = async (amountIn: string, amountOut: string) => {
+		const swapResult = await $canisters.icpswap.swap({
+			amountIn: amountIn.toString(),
```

Comment:
Here you don't deduct the transfer fee but you do in `depositInFastUnstake`. Try to make it consistent, either way is fine.